### PR TITLE
Fixing Italics naming + "notification" + "sign in"

### PR DESCRIPTION
**Pull request by anikbrazeau**
Fixes https://github.com/cds-snc/notification-api/issues/902

Removing all italics tags around the name.
Emphasizing the name by adding GC in front instead.

_Notify | Notification_  --->  GC Notify | GC Notification

--

Remove the use of notifications elsewhere to avoid confusion
Other than French name: notification ---> message

--

Change Log in ---> Sign in as per data/discussion in https://github.com/cds-snc/notification-admin/pull/585

### CODE REVIEW

File: app/templates/main_template.html [hunk 1]
```diff
@@ -171,7 +171,7 @@
                 <div
                   class="leading-none lg:ml-4 mr-5 lg:pl-5 font-semibold inline-block top-0 text-brand pb-0"
                 >
-                {{ _("<i>Notify</i>") }}
+                {{ _("Notify") }}
```
Comment (sastels) [inline app/templates/main_template.html:174]: Did you mean "GC Notify" here?

Comment (anikbrazeau) [reply app/templates/main_template.html:174]: We decided to switch all instances except the header where it becomes redundant as it sits next to the GoC logo.

File: app/assets/stylesheets/homepage.css [hunk 2]
```diff
@@ -1 +1,609 @@
...
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (sastels) [inline app/assets/stylesheets/homepage.css:1]: why did this file change? looks like reformatting?

File: app/templates/views/signedout.html [hunk 3]
```diff
@@ -33,16 +33,7 @@
...
-<div id="skiplink-container">
-  <div>
-    <a
-      href="#content"
-      class="skiplink"
-      >{{ skip_link_message|default('Skip to main content') }}</a
-    >
-  </div>
-</div>
+{% block page_title %} {{ _('GC Notify') }} {% endblock %} {% block main_content %}
```
[the remark this answers is not in the document]
Comment (anikbrazeau) [reply app/templates/views/signedout.html:36]: No? Maybe this PR pre-dates that fix?

File: app/assets/stylesheets/homepage.css (hunk 2 above)
Comment (anikbrazeau) [reply app/assets/stylesheets/homepage.css:1]: Not an intentional change here

File: app/main/forms.py [hunk 4]
```diff
@@ -1079,8 +1079,8 @@ class SetEmailBranding(StripWhitespaceForm):
         ]
     )
 
-    DEFAULT_EN = (FieldWithLanguageOptions.ENGLISH_OPTION_VALUE, 'English Government of Canada signature')
-    DEFAULT_FR = (FieldWithLanguageOptions.FRENCH_OPTION_VALUE, 'French Government of Canada signature')
+    DEFAULT_EN = (FieldWithLanguageOptions.ENGLISH_OPTION_VALUE, 'English Federal Identity Program (FIP)')
```
Comment (brdunfield) [inline app/main/forms.py:1082]: Was this an intentional revert?

File: app/main/forms.py [hunk 5]
```diff
@@ -1399,10 +1399,10 @@ def __init__(self, *args, **kwargs):
 
 
 branding_options = (
-    ('fip_english', 'English Government of Canada signature'),
-    ('fip_french', 'French Government of Canada signature'),
-    ('both_english', 'English Government of Canada signature and custom logo'),
-    ('both_french', 'French Government of Canada signature and custom logo'),
+    ('fip_english', 'Federal Identity Program (FIP) English only'),
```
Comment (brdunfield) [inline app/main/forms.py:1402]: Same here

File: app/templates/views/service-settings.html [hunk 6]
```diff
@@ -331,7 +331,7 @@ <h2 class="heading-medium">{{ _('Platform admin settings') }}</h2>
           {% call row() %}
             {% set txt = _('Email branding') %}
             {{ text_field(txt)}}
-            {{ text_field(_(current_service.email_branding_name)) }}
+            {{ text_field(current_service.email_branding_name) }}
```
Comment (brdunfield) [inline app/templates/views/service-settings.html:334]: Why did we remove the wrapping here? I think these translate the default branding names (..."Government of Canada signature")

File: app/templates/views/service-settings/request-to-go-live.html [hunk 7]
```diff
@@ -51,7 +51,7 @@
         {% if show_agreement %}
           {{ task_list_item(
             agreement_signed,
-            'Accept our data sharing and financial agreement',
+            'Accept our GC Notify data sharing and financial agreement',
```
Comment (brdunfield) [inline app/templates/views/service-settings/request-to-go-live.html:54]: should this be "the GC Notify"? (this is another "i'm not a content designer so feel free to ignore" comment)

File: app/templates/views/signedout.html [hunk 8]
```diff
@@ -286,7 +286,7 @@ <h2 class="leading-tight font-bold">
          {% endif %}
          </div>
           <ul class="list-disc ml-5">
-            <li class="pt-4">
+            <li class="pt-10 lg:pt-0 flex">
```
Comment (brdunfield) [inline app/templates/views/signedout.html:289]: Why was this changed?

File: app/templates/views/signedout.html [hunk 9]
```diff
@@ -460,16 +460,16 @@ <h2 class="leading-tight font-bold">
...
             <p class="pt-4">
-              {{ _("<i>Notify</i> supports roles and permissions, so that you can adjust which staff have access to send or edit messages.") }}
+              {{ _("GC Notify supports roles and permissions, so that you can adjust which staff have access to send or edit messages.") }}
             </p>
             <p class="pt-10 text-blue">
               <a
                 class="line-under inline-block text-blue text-base underline hover:no-underline"
                 href="{{ url_for('main.documentation') }}"
-                >{{ _("Learn how to integrate <i>Notify</i>") }}</a
+                >{{ _("Learn how to integrate GC Notify</i>") }}</a
```
Comment (brdunfield) [inline app/templates/views/signedout.html:472]: `</i>` slipped in!

File: tests/app/main/views/organisations/test_organisation.py [hunk 10]
```diff
@@ -257,9 +257,9 @@ def test_organisation_settings_for_platform_admin(
         'Name Org 1 Change',
         'Sector Not set Change',
         'Crown organisation Yes Change',
-        'Data sharing and financial agreement Not signed Change',
+        'GC Notify data sharing and financial agreement Not signed Change',
         'Request to go live notes None Change',
-        'Default email branding English Government of Canada signature Change',
+        'Default email branding English Federal Identity Program (FIP) Change',
```
Comment (brdunfield) [inline tests/app/main/views/organisations/test_organisation.py:262]: More old FIP strings

File: tests/app/main/views/test_service_settings.py [hunk 11]
```diff
@@ -66,7 +66,7 @@ def mock_get_service_settings_page_common(
         'Label Value Action',
         'Send emails On Change',
         'Reply-to email addresses Not set Manage',
-        'Email branding English Government of Canada signature Change',
+        'Email branding English Federal Identity Program (FIP) Change',
```
Comment (brdunfield) [inline tests/app/main/views/test_service_settings.py:69]: More old FIP

File: tests/app/main/views/test_template_folders.py [hunk 12]
```diff
@@ -550,6 +550,9 @@ def test_get_manage_folder_viewing_permissions_for_users_not_visible_for_service
         template_folder_id=folder_id,
         _test_page_title=False,
     )
+
+# SJA
```
Comment (brdunfield) [inline tests/app/main/views/test_template_folders.py:554]: What does this mean?

File: app/templates/views/signedout.html (hunk 9 above)
Comment (anikbrazeau) [reply app/templates/views/signedout.html]: Resolved

File: app/templates/views/agreement/agreement.html [hunk 13]
```diff
@@ -2,7 +2,7 @@
 {% from "components/sub-navigation.html" import sub_navigation %}
 
 {% block per_page_title %}
- {{ _('<i>Notify</i> data sharing and financial agreement') }}
+ {{ _('GCNotify data sharing and financial agreement') }}
```
[the remark this answers is not in the document]
Comment (anikbrazeau) [reply app/templates/views/agreement/agreement.html]: Resolved

File: app/templates/views/agreement/_agreement.html [hunk 14]
```diff
@@ -1,8 +1,8 @@
 <p>
-  {{ _('To use <i>Notify</i> your organisation') }} ({{ current_service.organisation.name }}) {{ _('must accept the <i>Notify</i> data sharing and financial agreement.') }}
+  {{ _('To use GC Notify= your organisation') }} ({{ current_service.organisation.name }}) {{ _('must accept the GC Notify data sharing and financial agreement.') }}
```
[the remark this answers is not in the document]
Comment (anikbrazeau) [reply app/templates/views/agreement/_agreement.html]: Resolved

File: app/templates/views/agreement/_agreement-choose.html [hunk 15]
```diff
@@ -1,5 +1,5 @@
 <p>
-  {{ _('Before you can go live on <i>Notify</i>, your organisation needs to accept to our data sharing and financial agreement.') }}
+  {{ _('Before you can go live, your organisation needs to accept to our GC Notify data sharing and financial agreement.') }}
```
[the remark this answers is not in the document]
Comment (anikbrazeau) [reply app/templates/views/agreement/_agreement-choose.html:2]: Resolved

File: app/templates/views/agreement/service-agreement-choose.html [hunk 16]
```diff
@@ -2,22 +2,22 @@
...
     {{ page_header(
       txt,
       back_link=url_for('main.request_to_go_live', service_id=current_service.id)
     )}}
 
     <p>
-      {{ _('Before you can go live on Notification, your organisation needs to
-      accept to our data sharing and financial agreement.') }}
+      {{ _('Before you can go live on GC Notify, your organisation needs to
+      accept to our GC Notify data sharing and financial agreement.') }}
```
[the remark this answers is not in the document]
Comment (anikbrazeau) [reply app/templates/views/agreement/service-agreement-choose.html]: resolved

File: app/main/views/send.py [hunk 17]
```diff
@@ -273,7 +273,7 @@ def s3_send(service_id, template_id):
             ))
         except (XLDateError):
             flash((
-                '{} contains numbers or dates that Notify can’t understand. '
+                '{} contains numbers or dates that GC Notify can’t understand. '
```
[the remark this answers is not in the document]
Comment (anikbrazeau) [reply app/main/views/send.py:276]: This seems to show up in the CSV?

File: app/templates/views/signedout.html (hunk 8 above)
Comment (anikbrazeau) [reply app/templates/views/signedout.html:289]: Not sure this was intentional?

File: app/templates/views/service-settings.html (hunk 6 above)
Comment (anikbrazeau) [reply app/templates/views/service-settings.html:334]: Indeed! :) we had switched this to https://github.com/cds-snc/notification-api/issues/848#issuecomment-642106153

English Government of Canada signature
French Government of Canada signature

Signature du gouvernement du Canada en anglais
Signature du gouvernement du Canada en français

File: tests/app/main/views/test_template_folders.py (hunk 12 above)
Comment (sastels) [reply tests/app/main/views/test_template_folders.py:554]: opps, forgot to take that out!

File: app/templates/admin_template.html [hunk 18]
```diff
@@ -21,7 +21,7 @@
 {% endblock %}
 
 {% block page_title %}
-   {% block per_page_title %}{% endblock %} – {{ _('Notification') }}
+   {% block per_page_title %}{% endblock %} – {{ _('Notify') }}
```
[the remark this answers is not in the document]
Comment (sastels) [reply app/templates/admin_template.html:24]: We have commented out IE6, IE7, etc code in several files - I suggest we delete it all in its own PR.

File: app/templates/views/service-settings.html (hunk 6 above)
Comment (brdunfield) [reply app/templates/views/service-settings.html:334]: I still think this needs to be wrapped, to make sure it gets translated. @sastels ?

File: app/templates/views/service-settings.html [hunk 19]
```diff
@@ -124,7 +124,7 @@ <h1 class="heading-medium">{{ _('Settings') }}</h1>
           {% set txt = _('Email branding') %}
           {{ text_field(txt) }}
           {% set txt = _('Your branding<br> ({})') %}
-          {{ text_field(txt.format(current_service.email_branding_name)|safe if current_service.email_branding else _(current_service.email_branding_name)) }}
+          {{ text_field(txt.format(current_service.email_branding_name)|safe if current_service.email_branding else current_service.email_branding_name) }}
```
Comment (brdunfield) [inline app/templates/views/service-settings.html:127]: Also missing wrapping here

File: app/templates/views/service-settings.html (hunk 6 above)
Comment (sastels) [reply app/templates/views/service-settings.html:334]: Yeah they need to be wrapped, and also the translations ☝️  aren't in the csv files so I'll add them too

File: app/translations/csv/fr.csv [hunk 20]
```diff
@@ -651,83 +653,82 @@
...
+"show how it is working through our <a href=""https://www.gov.uk/performance/govuk-notify"">performance</a> and <a href=""https://status.notifications.service.gov.uk/"">status</a> pages.","vous montrer comment le produit fonctionne dans les pages suivantes : <a href=""https://www.gov.uk/performance/govuk-notify"">Performance</a> et <a href= ""https://status.notifications.service.gov.uk/ "">État</a>. "
 "keep your data <a href=""/features/security"">secure</a>.","assurer la <a href=""/fonctionalités/sécurité"">sécurité</a> de vos données."
 "give you one month’s notice by email if we change our terms of use or delivery providers.","vous envoyer un préavis d’un mois par courriel en cas de modification des conditions d’utilisation ou des fournisseurs."
-"Leaving <i>Notify</i>","Quitter <i>Notification</i>"
-"You can leave <i>Notify</i> at any time. Just","Vous pouvez quitter <i>Notification</i> en tout temps. Vous n'avez qu’à"
+"Leaving GC Notify","Quitter Notification"
```
Comment (brdunfield) [inline app/translations/csv/fr.csv:711]: Should this be "Quitter GC Notification"?

File: app/translations/csv/fr.csv [hunk 21]
```diff
@@ -450,21 +449,21 @@
...
 "seen_cookie_message","message_témoin_vu"
 "Saves a message to let us know that you have seen our cookie message","Enregistre un message pour nous laisser savoir que vous avez vu notre message témoin"
 "1 month","1 mois"
-"Integrate the <i>Notify</i> API to send messages automatically.","Vous pouvez intégrer l’API <i>Notification</i> afin d'envoyer des messages automatiquement."
+"Integrate the API to send messages automatically.","Vous pouvez intégrer l’API afin d'envoyer des messages automatiquement."
 "Set up the client","Configurer le client"
 "The API is available using clients developed by <a href=https://www.notifications.service.gov.uk/ >GOV.UK</a>.","L'API est disponible en utilisant des clients développés par <a href=https://www.notifications.service.gov.uk/ >GOV.UK</a>."
 "À noter : ",""
 "Avant d'intégrer l'API avec les clients de Notification ci-dessous, vous devez modifier votre point de terminaison vers <a href=https://api.notification.alpha.canada.ca>https://api.notification.alpha.canada.ca</a>.",""
```
Comment (brdunfield) [inline app/translations/csv/fr.csv:463]: Should this also be "GC Notification"

File: app/translations/csv/fr.csv [hunk 22]
```diff
@@ -651,83 +653,82 @@
...
 "Send unlimited emails, completely free.","Envoyer un nombre illimité de courriels, tout à fait gratuitement."
-"<i>Notify</i> is a service managed by the Canadian Digital Service — no external procurement.","<i>Notification</i> est un service géré par le Service numérique canadien, sans autres achats externes."
-"Notify by the numbers","Notification en chiffres"
+"GC Notify is a service managed by the Canadian Digital Service — no external procurement.","GC Notification est un service géré par le Service numérique canadien, sans autres achats externes."
+"GC Notify by the numbers","GC Notification en chiffres"
 "Since November 2019","Depuis novembre 2019"
-"services using Notify","services qui utilisent Notification"
-"email notifications sent","notifications envoyées"
-"Try <i>Notify</i> now","Essayer <i>Notification</i> maintenant"
+"services using GC Notify","services qui utilisent Notification"
```
Comment (brdunfield) [inline app/translations/csv/fr.csv:684]: Another missing "GC"

File: app/translations/csv/fr.csv (hunk 21 above)
Comment (anikbrazeau) [reply app/translations/csv/fr.csv:463]: Fixed! 👍
